sencha-workspace/SlateAdmin/app/controller/people/Progress.js

```diff
@@ -271,7 +271,7 @@ Ext.define('SlateAdmin.controller.people.Progress', {
         var reportsStore = Ext.getStore('people.ProgressReports'),
             reportsProxy = reportsStore.getProxy();
 
-        reportsProxy.setExtraParam('termID', newValue);
+        reportsProxy.setExtraParam('TermID', newValue);
```

Comment:
the prevailing pattern is to keep query parameters lowercase and use ctype_digit to determine if they are IDs or handles rather than suffixing ID, so `?term=123` or `?term=y2016`